Retrovizor.Web/Controllers/ExamController.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Retrovizor.Data.Entities.Models;
+using Retrovizor.Domain.Repositories.Interfaces;
+
+namespace Retrovizor.Web.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ExamController : ControllerBase
```

Comment:
Tribala bi postojat metoda koja bi racunala postotak prolaznosti kod kategorija ispita (prva pomoc, voznja, propisi...).

Reply:
Nemamo primjenu za to trenutno, zaboravija san prokomentirat danas. Lako cemo posli dodat al sad doslovno nemamo to di stavit na UI bar koliko ja znan.

Reply:
Pogledaj dizajn za nove screen-ove sta je Bendic posla, ubacija je cini mi se na profil autoskole

Reply:
dodano.